app/views/patient_list_download_mailer/patient_list.html.erb

```diff
@@ -17,4 +17,4 @@
 <div>&nbsp;</div>
 <div>&nbsp;</div>
 
-<div style="font-size:110%;"><%= t("patient_list_email.unrequested").html_safe %></div>
+<div style="font-size:110%;"><%= t("patient_list_email.unrequested_html") %></div>
```

Comment:
This will require the locale file to be updated too. In the interest of time, I've pushed the change to your branch already 👍 

**Before**
<img width="641" alt="Screenshot 2020-03-23 at 12 36 05 PM" src="https://user-images.githubusercontent.com/4241399/77290779-e48c8d00-6d02-11ea-800d-29c4e5bbfe4c.png">


**After**
<img width="647" alt="Screenshot 2020-03-23 at 12 36 08 PM" src="https://user-images.githubusercontent.com/4241399/77290783-e6eee700-6d02-11ea-9a54-31304390a836.png">